JavaScript/CodingConventions.md

```diff
@@ -458,9 +479,10 @@ Comment early and often! For comments inside functions, use inline comments. For
 
 ```javascript
 /**
- * @desc This function bakes cookies
```

Comment:
AFAIK according to http://usejsdoc.org/tags-description.html the only readon to put `@description` (abbrevaited as `@desc`) would be to have possibility to move it around. It also makes JSDoc less readable.